src/components/address-bar/util.tsx

```diff
@@ -1,3 +1,8 @@
 export const routeWithApp = (route, app) => {
-  return `/${[route, app].filter(Boolean).join('/')}`;
```

Comment:
The initial version seems more readable.

Reply:
because we introduced a plugin to make it array items in multiple lines

Reply:
yeah. i think this one could be cleaned up just by not having it all inline in a string, probably.